modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/services/liferay/object/oppportunities/useGetOpportunities.ts

```diff
@@ -11,12 +11,14 @@ import {LiferayAPIs} from '../../common/enums/apis';
 import LiferayItems from '../../common/interfaces/liferayItems';
 import liferayFetcher from '../../common/utils/fetcher';
 
-export default function useGetOpportunities(pageSize: number) {
+export default function useGetOpportunities(parameters?: string) {
```

Comment:
```suggestion
export default function useGetOpportunities(parameters?: string) {
```

Esse hook eh utilizado em outros lugares? Chegou a testar se isso nao quebra em outros lugares que eh usado?

Reply:
so e usado em um unico lugar